dev/com.ibm.ws.kernel.boot.nested/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops

```diff
@@ -740,3 +740,5 @@ info.initlogs=Redirecting stdout and stderr to file {0}
 info.no.servers.defined=There are no servers defined in the user directory {0}.
 info.list.of.defined.servers=The following servers are defined relative to the user directory {0}.
 info.communicate.server=A communication error has occurred between the command {0} and the server {1}.
+
+info.serialFilterLoaded=Serial Filter Agent is loaded.
```

Comment:
Suggestion: The serial filter agent loaded.

Reply:
Hi Charlotte, got it. I'll update the text. Thank you for your review.

Reply:
@toshiyamamoto Great! After the updates are made, just let me know, and I'll add the ID reviewed tag. Thanks!

Reply:
I just updated the four files (I missed updating one files before..), and pushed. Thank you again.